src/esp/assets/ResourceManager.h

```diff
@@ -84,6 +87,13 @@ class ResourceManager {
   /** @brief Convenience typedef for Importer class */
   using Importer = Mn::Trade::AbstractImporter;
 
+  struct VHACDParameters : VHACD::IVHACD::Parameters {
+    VHACDParameters() { m_oclAcceleration = false; }
```

Comment:
Any reason we don't want OCL acceleration? Should add comment as to why.

Reply:
Simple: it's broken :D Both @aclegg3 and me verified that. (Neither of us bothered to check *why*, though.)